src/extensions/filter-control/bootstrap-table-filter-control.js

```diff
@@ -190,8 +191,20 @@ const UtilsFilterControl = {
           cookies.push(cookie)
         }
       })
-      return cookies
     }
+    if (foundLocalStorage) {
+      for (var i = 0; i < foundLocalStorage.length; i++) {
+        let cookie = foundLocalStorage.key(i)
+        if (/./.test(cookie)) {
+          cookie = cookie.split('.').pop()
+        }
```

Comment:
It needs to be judged whether this is right or not @UtechtDustin @djhvscf .

Reply:
Looks and works good to me